Examples/SPECAN_TXP_LOADPULL/SPECAN_TXP_LOADPULL.cs

```diff
@@ -0,0 +1,173 @@
+            saCommonConfig.ReferenceLevel_dBm = 0.0;
+
+            saAutolevelConfig = SA.AutoLevelConfiguration.GetDefault();
+            saAutolevelConfig.Enabled = true;
+            
+            txpConfigurationSpecAn = SA.RFmxSpecAn.TxpConfiguration.GetDefault();
+            txpConfigurationSpecAn.Rbw_Hz = 20e6;
+        }
+
+        public void Run()
+        {
+            #region Create Sessions
+            FocusITunerBroker iTuner = new FocusITunerBroker();
+            iTuner.Initialize(tunerAddress, false, true);
+            NIRfsg nIRfsg = new NIRfsg(resourceName, false, false);
+            RFmxInstrMX instr = new RFmxInstrMX(resourceName, "");
+            RFmxSpecAnMX specAn = instr.GetSpecAnSignalConfiguration(signalStringSpecan);
+            #endregion
+
+            #region Configure Tuner
+            FocusTuner.ConfigureCommon(iTuner, commonConfiguration);
+            #endregion
+
+            #region Configure Generation
+            ConfigureInstrument(nIRfsg, sgInstrConfig);
+            Waveform waveform = LoadWaveformFromTDMS(filePath);
+            DownloadWaveform(nIRfsg, waveform);
+            ConfigureContinuousGeneration(nIRfsg, waveform);
+            nIRfsg.Initiate();
+            #endregion
+
+            #region Configure Analyzer
+            saAutolevelConfig.MeasurementInterval_s = waveform.BurstLength_s;
+            SA.RFmxInstr.ConfigureInstrument(instr, saInstrConfig);
+            SA.RFmxSpecAn.ConfigureCommon(specAn, saCommonConfig);
+            SA.RFmxSpecAn.ConfigureTxp(specAn, txpConfigurationSpecAn);
+            #endregion
+
+            #region Measure
+            foreach (var gamma in gammaSweep)
+            {
+                Console.WriteLine("\n--------------------- Tuning --------------------\n");
+                currentGamma = FocusTuner.MoveTunerPerGamma(iTuner, gamma)[0];
+                PrintTuneResults();
+
+                Console.WriteLine("\n--------------------- Results --------------------\n");
+                RFmxSpecAnMXMeasurementTypes[] specanMeasurements = new RFmxSpecAnMXMeasurementTypes[1] { RFmxSpecAnMXMeasurementTypes.Txp };
+                SA.RFmxSpecAn.SelectAndInitiateMeasurements(specAn, specanMeasurements, saAutolevelConfig, waveform.SignalBandwidth_Hz, false, "", resultStringSpecan);
+                txpResultsSpecAn = SA.RFmxSpecAn.FetchTxp(specAn, RFmxSpecAnMX.BuildResultString(resultStringSpecan));
+                PrintTxPResults();
+            }
+            #endregion
+            
+            AbortGeneration(nIRfsg);
+            CloseInstrument(nIRfsg);
+            FocusTuner.CloseTuner(iTuner);
+            specAn.Dispose();
+            specAn = null;
+            instr.Close();
+            instr = null;
+        }
+
+        #region Utilities
+        private Complex[] GetConstantVSWR(double vswr, int numberOfPoints)
+        {
+            List<Complex> gammaArray = new List<Complex>();
```

Comment:
Not a must change, but since you know the number of points, I believe it makes more sense to use an array here instead of a list.

Reply:
Use array instead. And remove using generic namespace.